lib/fa2/common/tzip12.datatypes.jsligo

```diff
@@ -0,0 +1,37 @@
+export type tokenMetadataData = {
+    token_id: nat,
+    token_info: map<string, bytes>
+};
+
+export type tokenMetadata = big_map<nat, tokenMetadataData>;
```

Comment:
If it's a type of `big_map<nat, tokenMetadata>` this one should be tokenMetadatas or tokenMetadataMap

Reply:
we have to follow the tzip standard with naming